test/modules/implicit/parseTwice/main.compopts

```diff
@@ -0,0 +1,7 @@
+                      # main.good
+SomeLibrary.chpl
```

Comment:
You probably know something I don't about the test system, but I don't see where the .good data would be for all these options. Is main.good reused for all of them?

Reply:
Yeah,  it is since main.chpl is the file being compiled.  I'm not 100% sure that the comment on line 1 is strictly required, but I tend to put it on blank lines to make clear that the intention of the line is "compile via no compiler options" rather than "this is a blank line, to be skipped."